.github/workflows/format.yml

```diff
@@ -46,8 +46,8 @@ jobs:
           HEAD_REF: ${{github.event.pull_request.head.ref}}
         if: ${{ github.event.pull_request.head.repo.full_name == github.repository && steps.diff.outputs.result != '' }}
         run: |
-          git config user.name "hatohakaraage"
-          git config user.email "hatohakaraage@example.com"
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
:memo: 
https://docs.github.com/ja/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address
> GitHubアカウントを2017年7月18日以降に作成した場合、GitHubが提供するno-replyメールアドレスは7桁のID番号とユーザ名をID+username@users.noreply.github.comという形式にしたものになります。

https://api.github.com/users/github-actions[bot]

```json
{
  "login": "github-actions[bot]",
  "id": 41898282,
  "node_id": "MDM6Qm90NDE4OTgyODI=",
  "avatar_url": "https://avatars.githubusercontent.com/in/15368?v=4",
  "gravatar_id": "",
  "url": "https://api.github.com/users/github-actions%5Bbot%5D",
  "html_url": "https://github.com/apps/github-actions",
  "followers_url": "https://api.github.com/users/github-actions%5Bbot%5D/followers",
  "following_url": "https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}",
  "gists_url": "https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}",
  "starred_url": "https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}",
  "subscriptions_url": "https://api.github.com/users/github-actions%5Bbot%5D/subscriptions",
  "organizations_url": "https://api.github.com/users/github-actions%5Bbot%5D/orgs",
  "repos_url": "https://api.github.com/users/github-actions%5Bbot%5D/repos",
  "events_url": "https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}",
  "received_events_url": "https://api.github.com/users/github-actions%5Bbot%5D/received_events",
  "type": "Bot",
  "site_admin": false,
  "name": null,
  "company": null,
  "blog": "",
  "location": null,
  "email": null,
  "hireable": null,
  "bio": null,
  "twitter_username": null,
  "public_repos": 0,
  "public_gists": 0,
  "followers": 0,
  "following": 0,
  "created_at": "2018-07-30T09:30:16Z",
  "updated_at": "2018-09-18T23:02:41Z"
}
```